ts/components/conversation/AddMentions.tsx

```diff
@@ -15,9 +15,9 @@ interface MentionProps {
 }
 
 const StyledMentionAnother = styled.span<{ inComposableElement?: boolean }>`
+  user-select: all !important;
```

Comment:
does it make it selectable on the leftpane msg preview and on the message details screen (right panel).
?

Reply:
Made sure mentions and text content are only selectable in the composition box and main message list